src/components/cli-tutor-mode/CliTutorMode.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'redux-bundler-react'
+
+// Components
+import { Modal, ModalBody, ModalActions } from '../modal/Modal'
+import CopyIcon from '../../icons/CopyIcon'
+import Button from '../button/Button'
+import Overlay from '../overlay/Overlay'
+import Shell from '../shell/Shell'
+import StrokeDownload from '../../icons/StrokeDownload'
+
+export const CliTutorialModal = ({ command, t, onLeave, className, downloadConfig, ...props }) => {
+  const onClickCopyToClipboard = async (command) => {
+    await navigator.clipboard.writeText(command)
+  }
+
+  return (
+    <Modal {...props} className={className} onCancel={onLeave} style={{ maxWidth: '40em' }}>
+      <ModalBody icon={CopyIcon}>
+        <p className='charcoal w-80 center'>
+          {t('cli.description')}
+        </p>
+        <p className='charcoal-muted w-90 center'>
+          {command.includes('ipfs config replace') ? t('cli.extraNotes') : ''}
```

Comment:
Please change this to `command === cliCmdKeys.UPDATE_IPFS_CONFIG` (you need to import cliCmdKeys of course).

We should have a single source of truth for the command, just to avoid changing it on the constant files and breaking on this file.

Reply:
Thanks for digging this one, I've added comparison with the full command from `cliCmdKeys` now